### PR TITLE
Resolve members with double profiles

### PR DESCRIPTION
**Pull request by NPDebs**
This PR fixes issue #504.
I merged the information about people with double profiles.

In addition, I noticed and corrected:
- an error in a member's **`expertise`** section.
- an omission of pronoun.
- a typo in `country` section.

Screenshot (344)
Screenshot (345)

### CODE REVIEW

**Comment by NPDebs** [comment]
Hi @AndreaSanchezTapia  & @bebatut.
Behold...a new PR is ready. 👀 Please review when you can. Thanks! 🙌

File: _data/people.yaml [hunk 1]
```diff
@@ -350,13 +347,19 @@ alexwlchan:
...
+    technology. Also, I'm very keen to help address educational inequity as well as learning more
+    about community building and the open science community!
   city: London
+  country: United Kingdom
+  expertise:
+  - Editing
+  - Legal
+  - Diversity issues
   first-name: Ali
   github: false
```
Comment (bebatut) [inline _data/people.yaml:364]: Can you remove this one there?

Comment (NPDebs) [reply _data/people.yaml:364]: Yes, I can.

Two things I don't understand in life:
1. How I missed that line.
2. Why Github doesn't notify me anymore.

**Comment by bebatut** [comment]
The building is failing. It seems there is an error:

image

**Comment by NPDebs** [comment]
> The building is failing. It seems there is an error:
> 
> image

Woah. Let's see what's on line 1527. 👀🤔

**Comment by NPDebs** [comment]
I found it. Yaml is so strict about indentation.

**Comment by NPDebs** [comment]
Please let me know if it works now. 🙏

**Comment by NPDebs** [comment]
Hi @bebatut. 👀👋🏼

**Comment by AndreaSanchezTapia** [comment]
The local build is working here :tada: 
@NPDebs did you check the name you left is referenced in other documents? For example, between `alexandra-holinski` and `alexholinski` you chose the latter, but ols-3-metadata.yaml still refers to `alexandra-holinski` :(

**Comment by AndreaSanchezTapia** [comment]
Actually, this is the reason one check is failing, take a look ;)

**Comment by NPDebs** [comment]
😲😲😲 No. I didn't check. Will do that right away!

Thanks, @AndreaSanchezTapia. And welcome back. 😉

**Comment by AndreaSanchezTapia** [comment]
Check again the tests that are failing, please. It's the same mistake in different files.

**Comment by NPDebs** [comment]
> alexholinski

Yes yes. I'm still the checking each name in all the files, one after the other. 🙏

**Comment by NPDebs** [comment]
@AndreaSanchezTapia We're looking good already. 😃

**Comment by AndreaSanchezTapia** [comment]
The failed tests are saying everything 😅 that's why I told you to take a look at them

```
Ran on 103 files!
- ./_site/posts/2021/09/07/ols-4-annoucement/index.html


  *  linking to internal hash #alexholinski that does not exist (line 698)
     <a href="/ols-4#alexholinski">Alexandra Holinski</a>
  *  linking to internal hash #alexholinski that does not exist (line 756)
     <a href="/ols-4#alexholinski">Alexandra Holinski</a>
```

**Comment by NPDebs** [comment]
> 

Thanks!! 🙌 

I found this file in the _sites folder...but git does not track it. So... 👀

**Comment by AndreaSanchezTapia** [comment]
ah, of course, it comes from 2021-09-07-ols-4-annoucement.md - Let me check

**Comment by AndreaSanchezTapia** [comment]
check the ols-4 metadata and around.

**Comment by NPDebs** [comment]
> check the ols-4 metadata and around.

On it!

**Comment by NPDebs** [comment]
Eureka! 😭❤️

**Comment by NPDebs** [comment]
Oh, Andrea, you genius.  Thank you! 😭🤝

**Comment by NPDebs** [comment]
👀

**Comment by AndreaSanchezTapia** [comment]
Haha @NPDebs I see you don't lose anything. 
I had made a tiny comment about juanjo's name so I did the tiny edit. I will merge this soon. :)

**Comment by AndreaSanchezTapia** [comment]
@NPDebs Juan José needs a `github: false` field, could you add it, please?

**Comment by NPDebs** [comment]
Oops. I must have missed that comment. 🙈 Thanks for giving Juanjo justice. 😅🙌🏼

**Pull request merged**